client/app/widgets/Comments/__tests__/Comments.spec.jsx

```diff
@@ -83,41 +82,34 @@ const component = <Comments formProps={formProps} />;
 
 describe('Comments', () => {
   beforeEach(() => {
-    axiosPostSpy = jest.spyOn(axios, 'post').mockImplementation(() => Promise.resolve({
+    jest.spyOn(axios, 'post').mockImplementation(() => Promise.resolve({
```

Comment:
There is also a nice [shorthand](https://jestjs.io/docs/en/mock-function-api#mockfnmockreturnvaluevalue) for this